app/controllers/scc_accounts_controller.rb

```diff
@@ -98,6 +101,10 @@ def bulk_subscribe
 
   private
 
+  def find_available_gpg_keys
+    @selectable_gpg_keys = ::Katello::GpgKey.where(organization: @scc_account.organization).collect { |p| [p.name, p.id] }.unshift ['None', nil]
+  end
+
```

Comment:
Open question: Would it make sense to move the allocation of `@selectable_gpg_keys` into the `edit`-method in a way that `find_available_gpg_keys()` returns the search result instead of setting a variable?

IMHO this would make it easier to see that, for edit, the `selectable_gpg_keys`-variable is available.

Reply:
I see your point, then I would need to add this in the `new`-method as well. The reason why I did it that way is, that there is a `find_organization`-method which works the same way, but is a `before_action`-method. I wanted the `find_available_gpg_keys` to be something similar but it does not work for the `new`-method, so the call had to go inside the methods.

Reply:
> then I would need to add this in the `new`-method as well

Ah, somehow I missed that. I guess this is the DRYest solution then :grin: